laa_crime_application_store_app/services/permissions.py

```diff
@@ -0,0 +1,39 @@
+from typing import List
+
+from sentry_sdk import capture_message
+
+from laa_crime_application_store_app.models.application_schema import Application
+
+
+class Permissions:
+    LOCKED_STATES = ("granted", "expired", "auto_grant")
```

Comment:
I think CRM7 states `further_info` and `provider_requested` are also locked (or maybe they're provider-editable is we think the provider could in theory amend and re-submit them in the future)